enterprise/backend/src/metabase_enterprise/data_editing/data_editing.clj

```diff
@@ -137,9 +137,12 @@
   "Given a :effect/row.modified diff, figure out what kind of mutation it was."
   [{:keys [before after]}]
   (case [(some? before) (some? after)]
-    [false true]  :event/rows.created
-    [true  true]  :event/rows.updated
-    [true  false] :event/rows.deleted
+    [false true]  {:single :event/row.created
+                   :bulk   :event/rows.created}
```

Comment:
Maybe we should YAGNI the bulk events for now?

Reply:
removed